lib/graph3d/DataGroup.js

```diff
@@ -0,0 +1,240 @@
+var DataSet  = require('../DataSet');
+var DataView = require('../DataView');
+var Point3d  = require('./Point3d');
+var Range    = require('./Range');
+
+
+function DataGroup() {
+  this.dataTable = null;  // The original data table
+}
+
+
+/**
+ * Initialize the data from the data table. Calculate minimum and maximum values
+ * and column index values
+ * @param {Array | DataSet | DataView} rawData The data containing the items for
+ *                                             the Graph.
+ * @param {Number}                     style   Style Number
+ */
+DataGroup.prototype.dataInitialize = function(graph3d, rawData, style) {
```

Comment:
Better use active voice with verb first for your function calls i.e. `initializeData` ?